Lib/test/test_regrtest.py

```diff
@@ -22,7 +22,7 @@
 import textwrap
 import unittest
 from test import support
-from test.support import os_helper, without_optimizer
+from test.support import os_helper, import_helper
```

Comment:
Usually module imports are on separate lines.
```suggestion
from test.support import import_helper
from test.support import os_helper
```